DashAI/front/src/components/results/RunsTable.jsx

```diff
@@ -192,7 +194,16 @@ function RunsTable({ experimentId }) {
     }
   }, [experimentId]);
   return (
-    <Paper sx={{ py: 4, px: 4, ml: 7 }}>
+    <Paper
+      sx={{
+        p: 4,
+        [theme.breakpoints.up("xs")]: { ml: 8 },
+        [theme.breakpoints.up("sm")]: { ml: 12 },
+        [theme.breakpoints.up("md")]: { ml: 20 },
+        [theme.breakpoints.up("lg")]: { width: "80vw", ml: 15 },
+        [theme.breakpoints.up("xl")]: { width: "80vw", ml: 10 },
+      }}
+    >
```

Comment:
Question: Is the margin left assigned from the left edge of the image? I think it would be better if it was with respect to the drawer.

Reply:
Yes, the margin left is assing from the edge.

I am trying to define the margin with respect to the drawer, and I think I managed to do it. But by modifying that, now i haven't been able to make the table take up the entire space.

Reply:
I think that Drawer is not the best component to use in this case (it is more intended to be the drawer where the navigation menu of the page is located) but List.

Suggestion: You could implement a view with two `Grid`, the one on the right with a small size where the `List` of experiments (`ListItems`) goes and the one on the left where the visualization goes. 
And then make the grid sizes variable according to the size of the page using the typical sm, md, lg, etc...